modules/overrides/userconfigurable/client/limits.go

```diff
@@ -0,0 +1,118 @@
+package client
+
+import (
+	"github.com/grafana/tempo/pkg/util/listtomap"
+)
+
+type Limits struct {
+	Forwarders *[]string `json:"forwarders,omitempty"`
```

Comment:
Is the pointer to the slice required to achieve the desired result of `omitempty`?  I thought that a slice could be treated as nil.

Reply:
There is a slight nuance: `*[]string` can express 3 states, namely not set (`null`), empty (`[]`) and full (`["a"]`).
Example: https://go.dev/play/p/v-bQU9BHIqg

While this usually doesn't matter, for the overrides we want to know when you _don't_ set the overrides because in that case we will use the override from the configmap.

We use this property here:
https://github.com/grafana/tempo/blob/21fe98e8d2e9b3c52b5fa7fab2d3d2c72fe3977f/modules/overrides/user_configurable_overrides.go#L188-L193

So if `forwarders` is not set (= nil) we pass on the call to the runtimemanager (configmap). If it is set to empty slice `[]` we just return that.